tgui/packages/tgui/interfaces/NtosRoboControl.js

```diff
@@ -0,0 +1,157 @@
+import { multiline } from 'common/string';
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { AnimatedNumber, Box, Button, Grid, LabeledList, ProgressBar, Section, Tabs } from '../components';
+
+export const NtosRoboControl = props => {
+  const { act, data } = useBackend(props);
+  const bots = data.bots || [];
+  const mules = data.mules || [];
+  const { autoReturn,
+    autoPickup,
+    reportDelivery = [],
+  } = data;
+  return (
+    <Section
+      title="Robot Control Console"
+      buttons={(
+        <Button
+          content="Eject ID"
+          onClick={() => act('ejectcard')} 
+        />
+      )}>
+      <Box>
+        Id Card: {data.id_owner}
+      </Box>
+      <Box>
+        Bots detected in range: {data.botcount} 
+      </Box>
+      <Section>
+        {bots.map(robot => (
+          <Grid
+            key={robot.name}>
+            <Grid.Column>
+              <Box
+                backgroundColor={robot.mule_check === 1? "#6e4b0e" : "#342963"}>
+                <Section
+                  title={robot.name}>
+                  <Box>
+                    Model: {robot.model}
+                  </Box>
+                  <Box>
+                    Location: {robot.locat}
+                  </Box>
+                  <Box>
+                    Status: {robot.mode}
+                  </Box>
+                  {robot.mule_check === 1 &&(
+                    <Box>
+                      Loaded Cargo: {mules.load ? (mules.load) : "N/A"}
+                    </Box>
+                  )}
+                </Section>
+              </Box>
+            </Grid.Column>
+            <Grid.Column>
+              <Box>
+                {robot.mule_check === 1 &&(
+                  mules.map(mulebot => (
+                    (robot.bot_ref === mulebot.mule_ref) &&(
+                      <Box
+                        key={mulebot.mule_ref}
+                        textAlign="center">
+                        <Button
+                          tooltip="Stop Moving."
```

Comment:
for the tooltips you have mixed ending with a period and not ending with a period you should pick one and be consistent

Reply:
Tooltips all have periods, button content all don't have periods.

Reply:
oops didnt catch that >w<